doc/sphinx/system_setup.rst

```diff
@@ -263,64 +263,135 @@ Best explained in an example::
     therm.set_langevin(kT=1.0, gamma=1.0, seed=41)
 
 As explained before the temperature is set as thermal energy :math:`k_\mathrm{B} T`.
-The Langevin thermostat consists of a friction and noise term coupled
-via the fluctuation-dissipation theorem. The friction term is a function
-of the particle velocities. By specifying the diffusion coefficient for
-the particle becomes
 
-.. math:: D = \frac{\text{temperature}}{\text{gamma}}.
+The Langevin thermostat is based on an extension of Newton's equation of motion to
 
-The relaxation time is given by :math:`\text{gamma}/\text{MASS}`, with
-``MASS`` the particle's mass.  For a more detailed explanation, refer to
-:cite:`grest86a`.  An anisotropic diffusion coefficient tensor is available to
-simulate anisotropic colloids (rods, etc.) properly. It can be enabled by the
-feature ``PARTICLE_ANISOTROPY``.
+.. math::  m_i \dot{v}_i(t) = f_i(\{x_j\},v_i,t) - \gamma v_i(t) + \sqrt{2\gamma k_B T} \eta_i(t).
+
+Here, :math:`f_i` are all deterministic forces from interactions, 
+:math:`\gamma` the friction coefficient and :math:`\eta` a random, "thermal" force. 
+The random force  mimics collisions of the particle with surrounding solvent molecules 
+at temperature :math:`T` and satisfies
+
+.. math:: <\eta(t)> = 0 , <\eta^\alpha_i(t)\eta^\beta_j(t')> = \delta_{\alpha\beta} \delta_{ij}\delta(t-t')
+
+(:math:`<\cdot>` denotes the ensemble average and :math:`\alpha,\beta` are spatial coordinates). 
+
+In the |es| implementation of the Langevin thermostat,
+the additional terms only enter in the force calculation.
+This reduces the accuracy of the Velocity Verlet integrator
+by one order in :math:`dt` because forces are now velocity dependent.
+
+The random process :math:`\eta(t)` is discretized by drawing an uncorrelated random number
+:math:`\overline{\eta}` for each component of all the particle forces. 
+The distribution of :math:`\overline{\eta}` is uniform and satisfies
+ 
+.. math:: <\overline{\eta}> = 0 , <\overline{\eta}\overline{\eta}> = 1/dt
 
 The keyword ``seed`` controls the state of the random number generator (Philox
 Counter-based RNG) and is required on first activation of the thermostat. It
 can be omitted in subsequent calls of ``set_langevin()``. It is the user's
 responsibility to decide whether the thermostat should be deterministic (by
 using a fixed seed) or not (by using a randomized seed).
 
+The diffusion coeffictient :math:`D` of the particle can be obtained by the Einstein-Smoluchowski relation
```

Comment:
We should discuss to what degree the plain Einstein-Smoluchowski relation is applicable to the gamma which is set in the thermostat. In my mind what is presented in the text only holds for ideal systems. Interactions in the system will modify the DIffusion coefficient -- otherwise there would never be a need to measure diffusion coefficients using the mean squared displacement or a green kubo formula in MD simulations.

Reply:
I agree with @jonaslandsgesell, the documentation should not make statements about the diffusion coefficient here.

Reply:
@jonaslandsgesell I would remove the mention of the diffusion coefficient entirely, as @fweik suggested.

Reply:
Hey Christoph. Why not removing it?yes, I mean for ideal (noninteracting ) particles one should observe D=kT/gamma_thermostat. So we state that gamma is the friction of the thermostat somewhere?

Reply:
We say that gamma is the bare friction coefficient for the Langevin equation and any physics beyond that should be the user's responsibility.

Reply:
Sounds good!